src/path-transformers.ts

```diff
@@ -44,6 +44,10 @@
 
 import * as path from 'path';
 
+export function posixifyPath(filepath: string): string {
```

Comment:
Done.  I went and made some additional simplifications to the path-transformers file.